community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -394,6 +394,10 @@
     public static final Setting<Boolean> multi_threaded_schema_index_population_enabled =
             setting( "unsupported.dbms.multi_threaded_schema_index_population_enabled", BOOLEAN, TRUE );
 
+    @Internal
+    public static final Setting<Boolean> enable_native_schema_index =
```

Comment:
We went for it since it's easier to manage, also in tests. Now that we've gotten all fixed and working we can investigate if it would be tricky to manage this if it were a toggle instead. Hang on.